configure.ac

```diff
@@ -299,7 +301,9 @@ if test "$enable_scripting" = "yes"; then
    AX_LUA_LIBS([
      AC_DEFINE([HAVE_SCRIPTING], [1], [Have support for scripting])
      LIBS="$LIBS $LUA_LIB"
+     SCRIPTING=true
    ], [
+     SCRIPTING=false
```

Comment:
We now have `HAVE_SCRIPTING` and `SCRIPTING` as defined symbols that both indicate the same thing - I'm not sure if we discussed this but can you use `HAVE_SCRIPTING` in your test code?

Reply:
nope, as far as I can tell, conditionals used in configure.ac need to be declared using am_conditional, which `SCRIPTING` is

Reply:
Ok, can you just remove `HAVE_SCRIPTING` and use `#if SCRIPTING` instead?

Reply:
I don't think so - `AM_CONDITIONAL` for `SCRIPTING` declares an automake conditional, only visible in automake files while `AC_DEFINE` for `HAVE_SCRIPTING` defines a C preprocessor macro. They can't replace each other. Similar [pattern](https://github.com/dlqs/frr/blob/cd3c3a98e279326a0fc69a0f0d7620abdf8b404f/configure.ac#L1939-L1940)